### PR TITLE
RS/YJ/Rule 23-5

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This rule hasn't been tested yet. I'll work with Juan to test out this rule. Thanks!

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section23/section23rule5.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    """Rule 5 of ASHRAE 90.1-2019 Appendix G Section 23 (Air-side)"""
+
+    def __init__(self):
+        super(Section23Rule5, self).__init__(
+            rmrs_used=produce_ruleset_model_instance(
+                USER=False, BASELINE_0=True, PROPOSED=False
+            ),
+            each_rule=Section23Rule5.RuleSetModelInstanceRule(),
+            index_rmr=BASELINE_0,
+            id="23-4",
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section23/section23rule5.py:33]: Should update id

File: rct229/rulesets/ashrae9012019/section23/section23rule5.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            description="For baseline systems 6 and 8, Fans in parallel VAV fan-powered boxes shall run as the first stage of heating before the reheat coil is energized.",
+            ruleset_section_title="HVAC - Airside",
+            standard_section="G3.1.3.14 Fan Power and Control (Systems 6 and 8)",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_descriptions/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section23/section23rule5.py:39]: Should this be at terminal level?

**Review by weilixu — COMMENTED** [review]
Comments need to addressed.

File: rct229/rulesets/ashrae9012019/section23/section23rule5.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section23/section23rule5.py]: Addressed

File: rct229/rulesets/ashrae9012019/section23/section23rule5.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section23/section23rule5.py]: Addressed

**Comment by yunjoonjung-PNNL** [comment]
@jugonzal07 

After fixing the 23-5 rule (all the issues from my code), I pulled your branch (RT/JG/ruletest_23_5) and realized that there were more changes than I expected. Should I close this PR and create a new branch/PR for this rule?

**Comment by jugonzal07** [comment]
Hi @yunjoonjung-PNNL ,

I'm unsure why there are so many diffs. Perhaps a new, clean clean PR would address the issues! Please feel free to do that.

Thanks,
-Juan

**Pull request closed without merging**